src/main/java/org/opentripplanner/middleware/controllers/api/TripHistoryController.java

```diff
@@ -46,20 +46,15 @@
      */
     public static String getTripRequests(Request request, Response response, TypedPersistence<TripRequest> tripRequest) {
 
-        Auth0UserProfile requestingUser = Auth0Connection.getUserFromRequest(request);
-        if (requestingUser == null) {
-            logMessageAndHalt(request, HttpStatus.FORBIDDEN_403, "Authorization required.");
-        }
-
         final String userId = HttpUtils.getParamFromRequest(request, USER_ID_PARAM_NAME, false);
+
         OtpUser user = Persistence.otpUsers.getById(userId);
         if (user == null) {
             logMessageAndHalt(request, HttpStatus.FORBIDDEN_403, "Unknown user.");
```

Comment:
Should this be 404?

Reply:
I think it should be 403, however, after reviewing I have removed this check as it replicates the check carried out by isValidUser.

Reply:
^ ^ I'll add this as a change to #17.